src/vue-router/router.js

```diff
@@ -99,3 +127,21 @@ function inAppRouteGuard (to, from, next) {
     ? next()
     : next(vueRoutes.login)
 }
+
+// doesn't allow to visit verification pages if account is verified
+// or requested for verification
+function verificationGuard (to, from, next) {
+  const kycLatestData = store.getters[vuexTypes.kycLatestData]
```

Comment:
are you sure `kycLatestData` loaded at the moment of execution of verificationGuard?

Reply:
Yes, it is loaded after logging in